packages/editor/src/browser/editor-preferences.ts

```diff
@@ -474,6 +475,11 @@ export const editorPreferenceSchema: PreferenceSchema = {
             'default': EDITOR_FONT_DEFAULTS.fontWeight,
             'description': 'Controls the editor\'s font weight.'
         },
+        'editor.defaultEncoding': {
```

Comment:
should be  `files.encoding` to better support vs code settings and VS code extension expectations.

Reply:
I see. It's fixed now.

Reply:
@svenefftinge Hi, Efftinge, could you please take a look at this?